xbmc/platform/android/CPUInfoAndroid.h

```diff
@@ -8,17 +8,26 @@
 
 #pragma once
 
+#include "utils/CPUInfo.h"
 #include "utils/Temperature.h"
 
-#include "platform/posix/CPUInfoPosix.h"
+#include "platform/posix/filesystem/PosixFile.h"
 
-class CCPUInfoAndroid : public CCPUInfoPosix
+#include <memory>
+
+using namespace XFILE;
+
+class CCPUInfoAndroid : public CCPUInfo
 {
 public:
   CCPUInfoAndroid();
   ~CCPUInfoAndroid() = default;
 
   bool SupportsCPUUsage() const override { return false; }
   int GetUsedPercentage() override { return 0; }
-  float GetCPUFrequency() override { return 0; }
+  float GetCPUFrequency() override;
```

Comment:
By overriding this you are actually removing functionality. You should passthrough to the CCPUInfoPosix method from this method if you want to do this. See the other comment I left.